test/e2e/config.go

```diff
@@ -14,7 +14,7 @@ var (
 	BB                = "quay.io/libpod/busybox:latest"
 	healthcheck       = "quay.io/libpod/alpine_healthcheck:latest"
 	ImageCacheDir     = "/tmp/podman/imagecachedir"
-	fedoraToolbox     = "registry.fedoraproject.org/f32/fedora-toolbox:latest"
+	fedoraToolbox     = "registry.fedoraproject.org/fedora-toolbox:latest"
```

Comment:
I am afraid of using a `latest` tag outside of our control. Can we pin it to a version?

@edsantiago WDYT?

Reply:
You already know what I think, and you said it :-)

Reply:
the alternatives seem worse to me though:

```
[f29/fedora-toolbox](https://registry.fedoraproject.org/repo/f29%2Ffedora-toolbox/tags)
[docker pull registry.fedoraproject.org/f29/fedora-toolbox](https://registry.fedoraproject.org/repo/f29%2Ffedora-toolbox/tags)
[f30/fedora-toolbox](https://registry.fedoraproject.org/repo/f30%2Ffedora-toolbox/tags)
[docker pull registry.fedoraproject.org/f30/fedora-toolbox](https://registry.fedoraproject.org/repo/f30%2Ffedora-toolbox/tags)
[f31/fedora-toolbox](https://registry.fedoraproject.org/repo/f31%2Ffedora-toolbox/tags)
[docker pull registry.fedoraproject.org/f31/fedora-toolbox](https://registry.fedoraproject.org/repo/f31%2Ffedora-toolbox/tags)
[f32/fedora-toolbox](https://registry.fedoraproject.org/repo/f32%2Ffedora-toolbox/tags)
[docker pull registry.fedoraproject.org/f32/fedora-toolbox](https://registry.fedoraproject.org/repo/f32%2Ffedora-toolbox/tags)
[f33/fedora-toolbox](https://registry.fedoraproject.org/repo/f33%2Ffedora-toolbox/tags)
[docker pull registry.fedoraproject.org/f33/fedora-toolbox](https://registry.fedoraproject.org/repo/f33%2Ffedora-toolbox/tags)
[f34/fedora-toolbox](https://registry.fedoraproject.org/repo/f34%2Ffedora-toolbox/tags)
[docker pull registry.fedoraproject.org/f34/fedora-toolbox](https://registry.fedoraproject.org/repo/f34%2Ffedora-toolbox/tags)
[fedora-toolbox](https://registry.fedoraproject.org/repo/fedora-toolbox/tags)
[docker pull registry.fedoraproject.org/fedora-toolbox](https://registry.fedoraproject.org/repo/fedora-toolbox/tags)
```

Not sure what is the status of that image, but it doesn't even provide a F35 version.  F34 will be EOL in one week.

Reply:
@HarryMichal, git indicates you added that image with commit a1e1a3af71ac.  Can you give us guidance on which image we should use?

Reply:
> @HarryMichal, git indicates you added that image with commit https://github.com/containers/podman/commit/a1e1a3af71ac1bb650412c3236fab835bdc21cdc. Can you give us guidance on which image we should use?

The proposed change is correct and you can replace the `latest` tag with a number that matches a released Fedora version. So a safe choice now would be `registry.fedoraproject.org/fedora-toolbox:35` or even `registry.fedoraproject.org/fedora-toolbox:36` but probably a bit more stable-ish version is desired here.

The background behind the change can be found in https://github.com/containers/toolbox/pull/615.

Reply:
thanks for the explanation.

Switched to `fedora-toolbox:36`

Reply:
Thank you, @HarryMichal !